lib/common/src/CssLength.php

```diff
@@ -77,7 +77,7 @@ final class CssLength
      */
     public function __construct($attrValue)
     {
-        if ((! isset($attrValue) || '' === $attrValue)) {
+        if (null === $attrValue) {
```

Comment:
This better corresponds with `CssLength` in the AMP validator JS code:

```js
    if (input === undefined) {
      this.isValid = true;
      return;
    }
```

https://github.com/ampproject/amphtml/blob/406f2883379377c98a14e645c8f42a9f7794bfa7/validator/js/engine/validator.js#L3949-L3952